grafana/resource_team_external_group_test.go

```diff
@@ -0,0 +1,62 @@
+package grafana
```

Comment:
```suggestion
// +build enterprise

package grafana
```

We've started adding this build flag to enterprise resources. We don't yet have CI running against an enterprise instance so tests will fail.